examples/bar/mac-addr

```diff
@@ -0,0 +1 @@
+e2:14:27:53:9f:9f
```

Comment:
what is `examples/bar` for?

Reply:
crap, some old files got added. Force pushed an update